primer/brand
# Fix default FAQ content styling

## Summary

Improves default styling consistency for FAQ answers, by adding additional CSS rules to account for monolith overrides.

🔗[ Storybook preview](https://primer-f150de387a-26139705.drafts.github.io/storybook/?path=/story/components-faq-features--with-prose)

## List of notable changes:



- adds default spacing for `p`, `ol`, 'ul` and more to match Prose. 
- adds examples of using Prose into Storybook
- removed children filtering logic from the Accordion content, to instead style children using CSS 

## What should reviewers focus on?



- Review [the storybook examples](https://primer-f150de387a-26139705.drafts.github.io/storybook/?path=/story/components-faq-features--with-prose) and check everything looks in order. 
- Text should always be muted, not default

## Steps to test:



1. Open [this storybook URL](https://primer-f150de387a-26139705.drafts.github.io/storybook/?path=/story/components-faq-features--with-prose)
1. Check the text is using muted colors
1. Check the links use our inlinelink colors

## Supporting resources (related issues, external links, etc):

- [Enterprise LP](https://github.com/enterprise?_features=site_enterprise_react)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

With Prose default styling

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2023-10-16 at 10 29 17](https://github.com/primer/brand/assets/13340707/03e6e2f1-8b44-404d-8eca-dcbdefaed8ea)

 </td>
<td valign="top">

![Screenshot 2023-10-16 at 10 29 46](https://github.com/primer/brand/assets/13340707/e6f9c2b1-9e8b-4330-ac14-52e93a59d441)


</td>
</tr>
</table>


Correct link color styles 

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2023-10-16 at 10 28 24](https://github.com/primer/brand/assets/13340707/e9805ae0-a467-45b1-b569-286620edc628)


 </td>
<td valign="top">



![Screenshot 2023-10-16 at 10 31 27](https://github.com/primer/brand/assets/13340707/2db59ebd-2eff-4bbd-8ee7-5546932b56d8)

</td>
</tr>
</table>
